specs/extras/linked-token.md

```diff
@@ -0,0 +1,206 @@
+# Linked Token Spec
+
+## Outline for Linked Token Bridge
+
+<aside>
+💡 Claim: GMP will not be production-ready, and we cannot recommend its usage, until all these problems are solved.
```

Comment:
I don't think this entire Claim section should be part of the spec. LinkedToken is already used on production.

Reply:
changed here 9b776c81